src/main/java/net/sf/rails/game/GameManager.java

```diff
@@ -253,12 +253,16 @@ public void configureFromXML(Tag tag) throws ConfigurationException {
                             skipFirstStockRound);
 
                 for (String ruleTagName : srTag.getChildren().keySet()) {
-                    if (ruleTagName.equals("NoSaleInFirstSR")) {
-                        setGameParameter(GameDef.Parm.NO_SALE_IN_FIRST_SR, true);
-                    } else if (ruleTagName.equals("NoSaleIfNotOperated")) {
-                        setGameParameter(GameDef.Parm.NO_SALE_IF_NOT_OPERATED, true);
-                    } else if (ruleTagName.equals("NoSaleOfJustBoughtShare")) {
-                        setGameParameter(GameDef.Parm.NO_SALE_OF_JUST_BOUGHT_CERT, true);
+                    switch ( ruleTagName ) {
```

Comment:
Use of the new `switch()` format that allows String `cases`

Reply:
Essentially you're writing here mapping logic between a `String` and an `GameDef.Parm` enum.
I think it would be better to move this mapping logic to a new 

```java
public Parm parse(String ruleTagName) {
   ...
}
```

method in the `Parm` enum.
Afterwards we could use it like: `setGameParameter(GameDef.Parm.parse(ruleTagName), true);`

Reply:
These are only examples of newer JDK 8 syntax changes, not actual code rewrites which this type of enum parsing would be :)